localization/languages/tr.json

```diff
@@ -138,37 +138,37 @@
         "settingsBlockScriptsToggle": "Scriptleri engelle",
         "settingsBlockImagesToggle": "Resimleri engelle",
         "settingsBlockedRequestCount": {
-            "unsafeHTML": null //missing translation
+            "unsafeHTML": "Min şimdiye kadar, <strong></strong> reklam ve takipçi engelledi."
         },
-        "settingsCustomizeFiltersLink": null, //missing translation
+        "settingsCustomizeFiltersLink": "Filtreleri özelleştir",
         "settingsAppearanceHeading": "Görünüm",
         "settingsEnableDarkMode": "Koyu modu etkinleştir:",
-        "settingsDarkModeNever": null, //"Never" missing translation
-        "settingsDarkModeNight": null, //"At night" missing translation
-        "settingsDarkModeAlways": null, //"Always" missing translation
+        "settingsDarkModeNever": "Asla",
+        "settingsDarkModeNight": "Geceleyin",
+        "settingsDarkModeAlways": "Her zaman",
         "settingsSiteThemeToggle": "Site temasını etkinleştir",
         "settingsAdditionalFeaturesHeading": "Ek Özellikler",
         "settingsUserscriptsToggle": "Kullanıcı betiklerini etkinleştir",
-        "settingsSeparateTitlebarToggle": null, //missing translation
-        "settingsOpenTabsInForegroundToggle": null, //missing translation
+        "settingsSeparateTitlebarToggle": "Ayrılmış başlık çubuğu kullan"
+        "settingsOpenTabsInForegroundToggle": "Bir bağlantıyı yeni sekmede açtığımda hemen o sekmeye geç"
```

Comment:
This is the wording used in Firefox, right?

Min uses a different description than Firefox for this, so can you make the translation match the English setting name in Min?